### DISCUSSION TITLE
Self hosted solutions open source?

### DISCUSSION THREAD

**Discussion by mnemosdev** [supabase/supabase#851] [Questions]
Hello, do you plan to release Supabase as a self hosted solution under some open source license?

**Comment by kiwicopple** [comment]
Hey @mnemosdev We have a docker setup here: https://github.com/supabase/supabase/tree/master/docker

We'll integrate this with our CLI (eta this month): https://github.com/supabase/cli
We're improving the docs now to include instructions :) https://github.com/supabase/supabase/compare/docs/reference_docs_sidebar#diff-9fa709b26439a0cc9d1c314de0afe98f95d694ee7b35c4084a47b5d6c9ddc6ab

Let us know if you need any help with the Docker set up!
